### DISCUSSION TITLE
Version Upgrade Number not reflected in the Console

### DISCUSSION THREAD

**Discussion by dheeban-k** [outline/outline#4602] [General] [answered]
First of all thanks @tommoor for the amazing product ! Outline is a great KB tool for us!

The issue I'm facing lately is, while I upgraded the wiki from 0.66.1. to 0.67.0-pre.1 the UI for the version shows it's still two version behind. [1]
[2] The docker image which I pulled and followed the upgrade procedures from here: https://hub.docker.com/r/outlinewiki/outline

Please help me if I'm missing something here, thanks in advance :)

References:- 
[1] 
<img width="290" alt="image" src="https://user-images.githubusercontent.com/101546615/209250479-e5a76071-d247-4954-92ee-2c299677f2d1.png">

[2]
<img width="1585" alt="image" src="https://user-images.githubusercontent.com/101546615/209250620-3a63a13b-d157-44c2-8853-2431e18920dc.png">

**Comment by tommoor** [comment, marked as answer]
This is a known issue with the pre-release, the version is still correct despite what's displayed.
